packages/react/src/components/card/card.react-template.tsx

```diff
@@ -23,26 +24,38 @@ export const reactCard: ComponentImplementation<Card<JSX.Element>> = {
             <div slot="interactions" className="dso-card-interactions">
               {interactions.map((interaction, index) => (
                 <div key={index} className="dso-card-interaction">
-                  <button
-                    type={interaction.type}
-                    id={interaction.id}
-                    className="dso-tertiary"
-                    disabled={interaction.disabled}
-                    aria-describedby={interaction.ariaDescribedby}
-                    aria-expanded={interaction.ariaExpanded}
-                    aria-haspopup={interaction.ariaHaspopup}
-                    aria-roledescription={interaction.ariaRoledescription}
-                  >
-                    {interaction.icon && !interaction.iconMode && <DsoIcon icon={interaction.icon.icon}></DsoIcon>}
-                    <span className={interaction.iconMode === "only" ? "sr-only" : undefined}>{interaction.label}</span>
-                    {interaction.icon && interaction.iconMode && <DsoIcon icon={interaction.icon.icon}></DsoIcon>}
-                  </button>
+                  {interaction.type === "button" && (
+                    <button
+                      type={interaction.type}
+                      id={interaction.id}
+                      className="dso-tertiary"
+                      disabled={interaction.disabled}
+                      aria-describedby={interaction.ariaDescribedby}
+                      aria-expanded={interaction.ariaExpanded}
+                      aria-haspopup={interaction.ariaHaspopup}
+                      aria-roledescription={interaction.ariaRoledescription}
+                    >
+                      {interaction.icon && !interaction.iconMode && <DsoIcon icon={interaction.icon.icon}></DsoIcon>}
+                      <span className={interaction.iconMode === "only" ? "sr-only" : undefined}>
+                        {interaction.label}
+                      </span>
+                      {interaction.icon && interaction.iconMode && <DsoIcon icon={interaction.icon.icon}></DsoIcon>}
+                    </button>
+                  )}
+                  {interaction.type === "toggletip" && (
```

Comment:
`template: ({ toggleTipTemplate }) => {...}`

Dan kun je daarna hier `&& toggletipTemplate(interaction)`

Reply:
Ah ja, dat is beter.